.github/workflows/post_release_version_bump.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           --changelog_path "./.github/CHANGELOG.md" --post_release
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
+        uses: peter-evans/create-pull-request@v7
```

Comment:
Are we sure this won't break the workflow?

Reply:
Sure, Sure we don't. I'm optimistic, though. I can undo the change if you think it is too risky.

Reply:
I'm ok either way. Let's see how it does next release.